data/ai/lua/ai_helper.lua

```diff
@@ -1056,6 +1056,20 @@ end
 
 --------- Unit related helper functions ----------
 
+function ai_helper.is_passive_leader(aspect_value, id)
+    if (aspect_value == 'yes') then return true end
+    if (aspect_value == 'no') then return false end
```

Comment:
In an ideal world we'd be doing `if type(aspect_value) == 'boolean' then return aspect_value end`.

Reply:
Yeah, I assume that this will change once we switch to the variant aspect type.